README.md

```diff
@@ -302,23 +317,28 @@ dependencies {
             // Uncomment the next line if you want to use RSASSA-PSS (PS256, PS384, PS512) algorithms:
             //'org.bouncycastle:bcprov-jdk15on:1.60',
             'io.jsonwebtoken:jjwt-jackson:0.11.2' // or 'io.jsonwebtoken:jjwt-gson:0.11.2' for gson
+    // Gradle 7+ doesn't support the compile and runtime configuration anymore. Please use implementation and runtimeOnly instead.
```

Comment:
This comment is not needed as this PR covers the change already.

or maybe mention that older versions of Gradle require something different? (I’m not sure the original intent)

Reply:
Thanks for the review!
Seems that the white-space/formatting changed in PRs without an intention. 
Also, I also think that the part you mentioned above is not needed. 
Will revert that part and try a new PR!